### DISCUSSION TITLE
kpack-cli

### DISCUSSION THREAD

**Discussion by mindovermiles262** [asdf-community/.github#45] [General]
Hello,

I've just written a plugin to download and install kpack-cli for asdf. I'd like to merge it into the asdf community repo

https://github.com/mindovermiles262/asdf-kpack-cli

Thank you

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
